server/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "dependencies": {
     "body-parser": "^1.18.3",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v3.3.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-getHistoricAlerts",
```

Comment:
Reminder to self to update this one https://github.com/bravetechnologycoop/brave-alert-lib/pull/15 is merged.

Reply:
Done.